src/views/Tabs/TabsPage.vue

```diff
@@ -9,6 +9,8 @@
             :icon="logoVue"
             class="site-logo"
             @click="$router.push({ name: 'Home' })"
+            tabindex="0"
+            @keyup.enter="$router.push({ name: 'Home' })"
```

Comment:
Does the click and keyboard navigation still work if we nest the icon into a button?
For example in the last example of this page: https://ionicframework.com/docs/api/icon#accessibility

Reply:
The ion-button seems to mess up the hover effect of the icon actually. I've tried this in a time-box and haven't been able to figure out how to get the hover effect working with the ion-button added on. Do you think we can stay with this strategy?